apps/arena/lib/arena/game_updater.ex

```diff
@@ -485,6 +488,38 @@ defmodule Arena.GameUpdater do
     |> Map.put(:aditional_info, entity |> Entities.maybe_add_custom_info())
   end
 
+  defp report_game_results(state, winner_id) do
+    results =
+      Enum.filter(state.game_state.client_to_player_map, fn {client_id, _player_id} ->
+        Enum.member?(state.clients, client_id)
+      end)
```

Comment:
you can also use `Map.take(state.game_state.client_to_player_map, state.clients)`

Reply:
Nice tip! [622cd6d](https://github.com/lambdaclass/mirra_backend/pull/529/commits/622cd6d101e20b92d86fde749ce12600b3598d30)